src/components/radio-group/radio-group.tsx

```diff
@@ -0,0 +1,138 @@
+import React = require('react');
+import {EventHandler, ReactElement, SyntheticEvent} from "react";
+import {observable, action} from 'mobx';
+import {observer} from 'mobx-react';
+const style = require('./radio-group.css');
+
+export interface RadioGroupProps {
+    children: JSX.Element[];
```

Comment:
Strange. Since you use it as in `class RadioGroup extends React.Component<RadioGroupProps, {}>` it should be fine.